files/phoneformat.js/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "phoneformat.js",
+  "repo": "albeebe/phoneformat.js",
+  "files": {
+    "basePath": "dist",
+    "include": ["phone-format.js", "phone-format.min.js", "phone-format.min.js.map"]
```

Comment:
@megawac .map yes, but phone-format.js not. `*.min.*` will include map files as well